detection_rules/rule.py

```diff
@@ -211,7 +211,7 @@ def _validate_eql(ecs_versions, indexes, query, name):
                 message = exc.error_msg
                 trailer = None
                 if "Unknown field" in message and beat_types:
-                    trailer = "\nTry adding event.module and event.dataset to specify beats module"
+                    trailer = "\nTry adding event.dataset to specify beats module"
```

Comment:
Does beats always leave this field empty now?
```suggestion
                    trailer = "\nTry adding event.module or event.dataset to specify beats module"
```

Reply:
I originally removed it to dissuade its use - but since it should only not be used in cloud rules, probably better to add it back in 👍